conda_smithy/templates/build_steps.sh.tmpl

```diff
@@ -27,6 +27,23 @@ conda-build:
  root-dir: ${FEEDSTOCK_ROOT}/build_artifacts
 
 CONDARC
+
+### rough workaround to avoid singularity
+### read-only defaults:
+{% if build_with_singularity %}
```

Comment:
I think this file is shared during builds for different service providers. I believe that you need to use a bash variable for this if switch.

Reply:
Yes, that's a good point. I still don't know how we should pass this variable --- maybe an extra config in main in configure_feedstock.py? I need to actually go through configure_feedstock.py to understand the logic, I simply made it work in reverse

I'd like to also tighten a few things before we take this up. Currently, the log will print PID info (username, local libs, etc.) and I'd like to figure out a way to silence those

Reply:
@hmaarrfk could you check the new logic? Added a config in build-locally, that would trigger a separate process (two files for use with only singularity) rather than control flow

Reply:
added an env variable implementation: `if [ ${SINGULARITY_BUILD:-"False"} == "True" ]; then `...